torch/_export/serde/serialize.py

```diff
@@ -489,24 +489,22 @@ def save_graph_state(self):
 
     def handle_placeholder(self, node: torch.fx.Node):
         assert node.op == "placeholder"
-        if isinstance(node.meta["val"], torch.Tensor):
-            graph_input = Argument.create(as_tensor=TensorArgument(name=node.name))
-            self.graph_state.tensor_values[node.name] = serialize_tensor_meta(
```

Comment:
where did this line go?

Reply:
it's in [serialize_tensor_output](https://fburl.com/code/eq4ak3nz)